packages/wonder-stuff-ci/src/get-mobile-release-logger.ts

```diff
@@ -0,0 +1,79 @@
+import * as winston from "winston";
+import * as gcpLogger from "@google-cloud/logging-winston";
+const {combine, timestamp, json, errors, printf, colorize} = winston.format;
+
+/**
+ * Winston logger used for generating to GCP for the mobile release process (release-raccoon).
+ * @param {string} projectId The GCP project ID to send logs to.
+ * @param {string} logName The name of the log to send logs to in GCP.
+ * @param {string} logLevel The minimum severity level to log.
+ * @param {boolean} enableConsoleLogs If true, logs will be sent to the console as well as GCP.
+ * @param {[key: string]: string} labels K/V metadata that will be add to the logs
+ * @returns {winston.Logger} logger set up to send logs to GCP.
+ */
+export const getMobileReleaseLogger = (
+    args = {
```

Comment:
Is there a way to create default values for a parameters and then the caller passes only one value i.e. `enableConsoleLogs` set to true but everything else I want default?

Reply:
Yes, as you saw with @jeresig's suggestion, [here][1] (adding this comment as a paper trail for anyone reading this review later).

[1]: https://github.com/Khan/wonder-stuff/pull/636/files#r1164180337